x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.test.tsx

```diff
@@ -76,31 +96,51 @@ describe('AlertsTable', () => {
     'data-test-subj': 'testTable',
   };
 
-  it('should support sorting', async () => {
-    const wrapper = mountWithIntl(<AlertsTable {...tableProps} />);
-    await act(async () => {
-      await nextTick();
+  beforeEach(() => {
+    alertsTableConfigurationRegistryMock.get.mockClear();
+    alertsTableConfigurationRegistryMock.has.mockClear();
+  });
+
+  describe('Alerts table UI', () => {
+    it('should support sorting', async () => {
+      const wrapper = mountWithIntl(<AlertsTable {...tableProps} />);
+      await act(async () => {
+        await nextTick();
+        wrapper.update();
+      });
+      wrapper.find('.euiDataGridHeaderCell__button').first().simulate('click');
       wrapper.update();
+      wrapper
+        .find(`[data-test-subj="dataGridHeaderCellActionGroup-${columns[0].id}"]`)
+        .first()
+        .simulate('click');
+      wrapper.find(`.euiListGroupItem__label[title="Sort A-Z"]`).simulate('click');
+      expect(fetchAlertsData.onSortChange).toHaveBeenCalledWith([
+        { direction: 'asc', id: 'kibana.alert.rule.name' },
+      ]);
+    });
+
+    it('should support pagination', async () => {
```

Comment:
I moved the original tests to react testing library because it helps to simplify the code.

Reply:
@chrisronline

Reply:
Okay sure!